src/Symfony/Component/Debug/Tests/BufferingLoggerTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Debug\Tests;
+
+use Psr\Log\LogLevel;
+use Symfony\Component\Debug\BufferingLogger;
+use Symfony\Component\Debug\ErrorHandler;
+
+/**
+ * BufferingLoggerTest.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class BufferingLoggerTest extends \PHPUnit_Framework_TestCase
+{
+    public function testFlush()
+    {
+        for ($strip = 0; $strip < 2; ++$strip) {
```

Comment:
use a dataProvider for that instead of a loop in the test. It will be much easier to debug failures as we will know which case is failing.